Dockerfile.distroless

```diff
@@ -0,0 +1,43 @@
+FROM golang:1.17-buster as builder
+
+WORKDIR /workspace
+RUN apt-get update && apt-get install --no-install-recommends -y git gcc g++ ca-certificates mime-support \
+  && git clone https://github.com/drakkan/sftpgo.git
+ARG TAG
+ARG FEATURES
+ENV GOOS=linux GO111MODULE=on GOPROXY="https://proxy.golang.org,direct" CGOENABLED=0 GOFLAGS="-mod=readonly"
+
+WORKDIR /workspace/sftpgo
+
+# Use --build-arg TAG=LATEST for latest tag. Use e.g. --build-arg TAG=v1.0.0 for a specific tag/commit. Otherwise HEAD (master) is built.
+RUN git checkout $(if [ "${TAG}" = LATEST ]; then echo `git rev-list --tags --max-count=1`; elif [ -n "${TAG}" ]; then echo "${TAG}"; else echo HEAD; fi)
+
+
+RUN go mod download
+
+RUN set -xe && \
+    export COMMIT_SHA=${COMMIT_SHA:-$(git describe --always --dirty)} && \
+    go build $(if [ -n "${FEATURES}" ]; then echo "-tags ${FEATURES}"; fi) -tags netgo -a -ldflags "-linkmode external -extldflags '-static' -s -w -X github.com/drakkan/sftpgo/version.commit=${COMMIT_SHA} -X github.com/drakkan/sftpgo/version.date=`date -u +%FT%TZ`" -v -o sftpgo
+
+# Modify the default configuration file
+RUN sed -i "s|\"users_base_dir\": \"\",|\"users_base_dir\": \"/srv/sftpgo/data\",|" /workspace/sftpgo/sftpgo.json && \
+    sed -i "s|\"backups\"|\"/srv/sftpgo/backups\"|" /workspace/sftpgo/sftpgo.json && \
+    sed -i "s|\"address\": \"127.0.0.1\",|\"address\": \"\",|" /workspace/sftpgo/sftpgo.json
+
+# use distroless/static-debian11 https://console.cloud.google.com/gcr/images/distroless/global/static-debian11@sha256:157bbd69d9af19adf370b0e05af074170a6788e218e8e3bc7c2763897a98890d/details
+FROM gcr.io/distroless/static-debian11@sha256:157bbd69d9af19adf370b0e05af074170a6788e218e8e3bc7c2763897a98890d
```

Comment:
we could use

```
FROM gcr.io/distroless/static-debian11
```

here, so we always get the latest stable version

Reply:
updated base image to `gcr.io/distroless/static-debian11`